README.md

```diff
@@ -37,6 +37,38 @@ pnpm i18n:get-translations
 
 # Builds and serves assets with hot-reload
 pnpm dev
+
+```
+
+### Using HTTPS Locally
+
+To enable SSL support in loval development, use the `pnpm dev:secure` command. This requires you to have a private key and certificate in the root of the repository with the following names:
+
+```shell
+localhost+1-key.pem # The private key file
+localhost+1.pem # The certificate file
+```
+
+The easiest way to create these files is with a local development tool called [mkcrt](https://github.com/FiloSottile/mkcert). After [installing mkcert](https://github.com/FiloSottile/mkcert#installation) and activating it with `mkcert -install`, run the following command in the root of the repository:
+
+```bash
+# Use `mkcert` to create a certificate for `localhost` and for the external IP address used by nuxt.
+# You may need to regenerate the certificate if this IP address changes for any reason.
+#
+# You can find the IP by looking at the output of `nuxt dev`. For example:
+#  ╭────────────────────────────────────────────╮
+#  │                                           │
+#  │   Nuxt @ v2.15.8                          │
+#  │                                           │
+#  │   ▸ Environment: development              │
+#  │   ▸ Rendering:   server-side              │
+#  │   ▸ Target:      server                   │
+#  │                                           │
+#  │   Listening: http://192.168.50.119:8443/  │ # <-- Use this IP Address
+#  │                                           │
+#  ╰────────────────────────────────────────────╯
```

Comment:
Why is this stuff as a comment instead of just in the markdown?

Reply:
No reason; fixed.